addons/web/static/tests/services/notification_service_tests.js

```diff
@@ -55,7 +55,11 @@ QUnit.module('Services', {
             "<div class=\"toast-header\"> <span class=\"fa fa-2x mr-3 fa-lightbulb-o o_notification_icon\" role=\"img\" aria-label=\"Notification undefined\" title=\"Notification undefined\"></span> <div class=\"d-flex align-items-center mr-auto font-weight-bold o_notification_title\">a</div> </div> <div class=\"toast-body\"> <div class=\"o_notification_content\">b</div> </div>",
             "should display notification");
         assert.containsNone($notification, '.o_notification_close', "should not display the close button in ");
-        await testUtils.nextTick();
+        // We need to use a delay above 0 ms because otherwise the notification will close right after it opens
+        // before we can perform any test.
```

Comment:
I would put this commit on top in the beforeEach section and I would define a function on top of the file that return a promise with the delay you set (1) (and that you can then use here)